content/en/real_user_monitoring/guide/upload-javascript-source-maps.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Upload Javascript source maps
+kind: guide
+further_reading:
+- link: '/real_user_monitoring/error_tracking'
+  tag: 'Documentation'
+  text: 'Get started with Error Tracking'
+- link: '/real_user_monitoring/error_tracking/explorer'
+  tag: 'Documentation'
+  text: 'Visualize your Error Tracking data in the Explorer'
+---
+
+## Overview
+
+If your front-end Javascript source code is minified, you will need to upload your source maps to Datadog so that we are able to deobfuscate your different stack traces: for a given error, you will then get access to the file path, the line number as well as a code snippet for each frame of the related stack trace.
```

Comment:
```suggestion
If your front-end Javascript source code is minified, you will need to upload your source maps to Datadog so that we are able to deobfuscate your different stack traces. For a given error, you will then get access to the file path, the line number, as well as a code snippet for each frame of the related stack trace.
```